--- damask-3.1.0.orig/src/CLI.f90
+++ damask-3.1.0/src/CLI.f90
@@ -327,7 +327,7 @@
 !--------------------------------------------------------------------------------------------------
 subroutine printCompileOptions() bind(C, name="F_printCompileOptions")
 
-  print'(1x,a)', 'Compiler options: '//compiler_options()
+  !print'(1x,a)', 'Compiler options: '//compiler_options()
   print'(1x,a)', 'Compiled for: '//CMAKE_SYSTEM_NAME//' on '//CMAKE_SYSTEM_PROCESSOR
 
 end subroutine printCompileOptions
